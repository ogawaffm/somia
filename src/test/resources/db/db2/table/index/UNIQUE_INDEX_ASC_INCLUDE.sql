DROP TABLE IF EXISTS TEST.UNIQUE_INDEX_ASC_INCLUDE
/
CREATE TABLE TEST.UNIQUE_INDEX_ASC_INCLUDE
(
    COL_1 VARCHAR(10) NOT NULL,
    COL_2 VARCHAR(10) NOT NULL
)
/
CREATE UNIQUE INDEX TEST.UNIQUE_IDX_ASC_INCLUDE ON TEST.UNIQUE_INDEX_ASC_INCLUDE (COL_1 ASC) INCLUDE (COL_2)
/