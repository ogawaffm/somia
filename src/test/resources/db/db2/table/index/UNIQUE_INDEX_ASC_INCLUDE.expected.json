{
  "name": "UNIQUE_INDEX_ASC_INCLUDE",
  "catalogName": "",
  "schemaName": "TEST",
  "typeName": "TABLE",
  "comment": "",
  "columns": [
    {
      "name": "COL_1",
      "type": {
        "name": "VARCHAR",
        "typeNumber": 12,
        "className": "java.lang.String",
        "precision": 10,
        "scale": 0,
        "isNullable": false,
        "isAutoIncrement": false,
        "numericPrecisionRadix": 0,
        "isCaseSensitive": false,
        "isSearchable": true,
        "isSigned": false,
        "isCurrency": false,
        "columnDisplaySize": 10,
        "baseType": null
      },
      "comment": "",
      "collation": "",
      "defaultDefinition": ""
    },
    {
      "name": "COL_2",
      "type": {
        "name": "VARCHAR",
        "typeNumber": 12,
        "className": "java.lang.String",
        "precision": 10,
        "scale": 0,
        "isNullable": false,
        "isAutoIncrement": false,
        "numericPrecisionRadix": 0,
        "isCaseSensitive": false,
        "isSearchable": true,
        "isSigned": false,
        "isCurrency": false,
        "columnDisplaySize": 10,
        "baseType": null
      },
      "comment": "",
      "collation": "",
      "defaultDefinition": ""
    }
  ],
  "indexes": [
    {
      "name": "",
      "columns": [],
      "isUnique": false,
      "comment": "",
      "qualifierName": "",
      "isClustered": false,
      "isHashed": false,
      "isStatistic": true,
      "whereClause": "",
      "cardinality": 0,
      "pages": 0
    },
    {
      "name": "UNIQUE_IDX_ASC_INCLUDE",
      "columns": [
        {
          "name": "COL_1",
          "order": "ASC",
          "isIncluded": false
        },
        {
          "name": "COL_2",
          "order": "",
          "isIncluded": false
        }
      ],
      "isUnique": true,
      "comment": "",
      "qualifierName": "TEST",
      "isClustered": false,
      "isHashed": false,
      "isStatistic": false,
      "whereClause": "",
      "cardinality": 0,
      "pages": 0
    }
  ],
  "collation": "",
  "primaryKey": null,
  "bestRowIdentifier": null
}